engines/gob/inter_v1.cpp

```diff
@@ -1230,9 +1271,24 @@ void Inter_v1::o1_keyFunc(OpFuncParams &params) {
 		_vm->_util->clearKeyBuf();
 		break;
 
+	case -1:
```

Comment:
May this change break other games? Because now `-1` is checks keys.

Reply:
You're right, I will revert to previous behaviour for non-Adibou games.